ui/utils/dimension.js

```diff
@@ -0,0 +1,43 @@
+import { useState, useEffect } from "react";
```

Comment:
Include a description of what of the purpose of this utility and an example use.